index.js

```diff
@@ -0,0 +1,236 @@
+const express = require('express');
+const server = express();
+const port = 9000;
+const userDb = require('./data/helpers/userDb.js');
+const postDb = require('./data/helpers/postDb.js');
+const tagDb = require('./data/helpers/tagDb.js');
+const cors = require('cors');
+const helmet = require('helmet');
+const morgan = require('morgan');
+
+server.use(cors());
+server.use(helmet());
+server.use(morgan('tiny'));
+server.use(express.json());
+
+const upperCase = (req, res, next) => {
+    req.name = req.body.name.toUpperCase();
+    console.log(req.name);
+    next();
+};
+
+server.get('/', (req, res) => {
+	res.send('<h2>Server is running.</h2>');
+})
+
+//// GETS USER LIST ////
+server.get('/api/users', (req, res) => {
+	userDb
+	.get()
+	.then(users => {
+		res.status(200).json(users)
+	})
+	.catch(error => {
+		console.log(error)
+		res.status(500).json({msg: "the users information could not be recieved"})
+	})
+})
+
+//// FIRST LETTER CAPITAL ////
+const middleUP = (req, res, next) => {
+	const {name} = req.body;
+	req.body.name = req.body.name.charAt(0).toUpperCase() + req.body.name.slice(1);
+	next()
+}
```

Comment:
Move this function to the top so it isn't randomly placed here between two GET endpoints.